controllers/goharbor/exporter/networkpolicies.go

```diff
@@ -0,0 +1,72 @@
+package exporter
+
+import (
+	"context"
+
+	goharborv1 "github.com/goharbor/harbor-operator/apis/goharbor.io/v1alpha3"
+	harbormetav1 "github.com/goharbor/harbor-operator/apis/meta/v1alpha1"
+	"github.com/goharbor/harbor-operator/pkg/graph"
+	"github.com/pkg/errors"
+	netv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+type NetworkPolicy graph.Resource
+
+func (r *Reconciler) AddNetworkPolicies(ctx context.Context, exporter *goharborv1.Exporter) error {
+	areNetworkPoliciesEnabled, err := r.AreNetworkPoliciesEnabled(ctx, exporter)
+	if err != nil {
+		return errors.Wrapf(err, "cannot get status")
```

Comment:
I suppose that you copy-pasted it from somewhere else, but I suggest to change this error message which does not match the previous call.

```suggestion
		return errors.Wrapf(err, "check network policy")
```

Pease tell me if it is out of the scope, it wan be ^^'

Reply:
Let's keep `return errors.Wrapf(err, "cannot get status")` same with other components.

If really need to change this error message, I will open a new PR to change this message in the whole source code.